airflow-core/src/airflow/ui/src/pages/Iframe.tsx

```diff
@@ -49,12 +49,38 @@ export const Iframe = ({ sandbox = "allow-same-origin allow-forms" }: { readonly
     return <ErrorPage />;
   }
 
+  // Build the href URL with context parameters if the view has a destination
+  let src = iframeView.href;
+
+  if (iframeView.destination !== undefined && iframeView.destination !== "nav") {
+    // Check if the href contains placeholders that need to be replaced
+    if (dagId !== undefined) {
+      src = src.replaceAll("{dag_id}", dagId);
+    }
+    if (runId !== undefined) {
+      src = src.replaceAll("{run_id}", runId);
+    }
+    if (taskId !== undefined) {
+      src = src.replaceAll("{task_id}", taskId);
```

Comment:
In the backend doc we have `{DAG_ID}, {RUN_ID}, etc...` as a pattern. Do we want to replace it with lower case `{dag_id}, {run_id}, etc...`?

Reply:
No, let me update the frontend